CMakeLists.txt

```diff
@@ -94,6 +94,12 @@ option (PIO_USE_MPISERIAL    "Enable mpi-serial support (instead of MPI)"   OFF)
 option (PIO_USE_PNETCDF_VARD       "Use pnetcdf put_vard "  OFF)
 option (WITH_PNETCDF         "Require the use of PnetCDF"                   ON)
 
+if(APPLE)
```

Comment:
@edhartnett do we need the same for the autoconf version?

Reply:
Yes, it looks like it does. In the linked issue he was using `./configure`. I've run into this issue in CMake before, and know the fix. Not sure how to translate it to Autotools.

Reply:
Yes, let's get this merged and then I'll add something to configure.ac whcih I hope will solve the problem.